code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -346,7 +346,7 @@
 	required_reagents = list("stable_plasma" = 1, "radium" = 1, "phosphorus" = 1)
 
 /datum/chemical_reaction/pyrosium/on_reaction(datum/reagents/holder, created_volume)
-	holder.chem_temp = 20 // also cools the fuck down
+	holder.chem_temp += 500 // also heats the fuck up AGAIN
```

Comment:
"holder.chem_temp = 20 // also cools the fuck down"

Does this make sense to you?